openmc/mgxs_library.py

```diff
@@ -380,34 +286,34 @@ def representation(self, representation):
         check_value('representation', representation, _REPRESENTATIONS)
         self._representation = representation
 
-    @alias.setter
-    def alias(self, alias):
-        if alias is not None:
-            check_type('alias', alias, basestring)
-            self._alias = alias
-        else:
-            self._alias = self._name
-
     @awr.setter
     def awr(self, awr):
         # Check validity of type and that the awr value is > 0
         check_type('awr', awr, Real)
         check_greater_than('awr', awr, 0.0)
         self._awr = awr
 
-    @kT.setter
-    def kT(self, kT):
-        # Check validity of type and that the kT value is >= 0
-        check_type('kT', kT, Real)
-        check_greater_than('kT', kT, 0.0, equality=True)
-        self._kT = kT
+    @fissionable.setter
+    def fissionable(self, fissionable):
```

Comment:
ditto per my earlier comment - cannot this be determined under-the-hood from the input data?